src/mongoc/mongoc-bulk-operation.c

```diff
@@ -370,6 +370,13 @@ mongoc_bulk_operation_set_write_concern (mongoc_bulk_operation_t      *bulk,
    }
 }
 
+const mongoc_write_concern_t *
+mongoc_bulk_operation_get_write_concern (mongoc_bulk_operation_t      *bulk)
```

Comment:
I don't think we need the extra whitespace here between mongoc_bulk_operation_t and *bulk.